limits.md

```diff
@@ -2,16 +2,15 @@
 
 In general, we currently allow
 
-* an unlimited number of applications
-* an unlimited number of services per application
+* an unlimited number of services
 * an unlimited number of components per service
 
 However, we enforce these restrictions:
 
-* any application older than 7 days (from creation time) will be deleted
+* any service older than 7 days (from creation time) will be deleted
   * except if you have configured any [custom domain name](https://docs.giantswarm.io/reference/swarm-json/#own-domain-name)
-* a maximum of 10 instances (containers) per component
-* a memory (RAM) limit of 512 MB per instance (container)
+* a maximum of 10 component instances (containers) per component
```

Comment:
Just to make clear in this context what they are? We also use the term "component instances" as a long form in a few places, to make very clear that instances are actually component instances.